### ISSUE TITLE
Issue about xml response and new feature suggestion

### ISSUE THREAD

**Issue by The-Earth** [mwclient/mwclient#195]
site.api() can't deal with responses in xml format such as responses to an export query. That's because site.api() always expects response in json format. (I have searched 'xml' in the documents and found nothing) Furthermore, it'll be awesome if mwclient supports Export and Import feature. (like what we can do in Special:Export and Special:Import)

**Comment by danmichaelo** [comment]
You're right, export is not really supported. You can still use raw_call to make a manual api call though:

    site.raw_call('api', {'action': 'query', 'titles': 'Page title', 'export': 1})
